bccsp/schemes/idemixopts.go

```diff
@@ -305,6 +320,17 @@ func (o *EidNymAuditOpts) HashFunc() crypto.Hash {
 	return 0
 }
 
+type RhAuditOpts struct {
+	AuditVerificationType AuditVerificationType
+	RhIndex               int
+	RevocationHandle      string
+	RRh                   *math.Zr
+}
+
+func (o *RhAuditOpts) HashFunc() crypto.Hash {
```

Comment:
Why is this needed?

Reply:
Not entirely sure what this function is used for, I added it because it exists for the enrollment ID. If there is no use for it then I can remove it.

Reply:
I just checked, the HashFunc is required to allow a type to implement the SignerOpts interface. There are multiple places in idemix where a SignerOpts parameter is type asserted to be either a EidNymAuditOpts or RhNymAuditOpts. I have not however found a call to the function so maybe a redesign of the interface is warranted.